attestation-agent/attestation-agent/src/bin/ttrpc-aa/ttrpc_protocol/attestation_agent.rs

```diff
@@ -9,7 +9,6 @@
 #![allow(unused_attributes)]
 #![cfg_attr(rustfmt, rustfmt::skip)]
 
-#![allow(box_pointers)]
```

Comment:
do we have a code path in CI that is not built since we did not catch this part earlier?

Reply:
I am afraid not. We should turn on `ttrpc` feature in the ci also to cover this. Let me do a fix in this PR also.